experimental/hip/tests/stablehlo_ops/CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/divide.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/dot.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/dot_bf16.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/dot_general.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/dynamic_slice.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/dynamic_update_slice.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/exponential.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/exponential_fp16.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/exponential_minus_one.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/fft.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/finite.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/floor.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/gather.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/householder.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/iota.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/log.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/log_plus_one.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/maximum.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/minimum.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/multiply.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/negate.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/pad.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/philox.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/pow.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/reduce.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/reduce_window.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/remainder.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/reshape.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/reverse.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/rng_normal.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/rng_uniform.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/round.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/rsqrt.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/scatter.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/scatter_dynamic.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/select.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/sine.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/slice.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/sort.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/sqrt.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/subtract.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/tanh.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/three_fry.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/torch_index_select.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/transpose.mlir"
+    "${IREE_SOURCE_DIR}/tests/e2e/stablehlo_ops/while.mlir"
+  TARGET_BACKEND
+    "rocm"
```

Comment:
Does this get past the checks here? https://github.com/openxla/iree/blob/d9b1fdc441d3e1e770e587785781724f36c120ca/build_tools/cmake/iree_check_test.cmake#L89-L104

(I've been thinking about removing those checks and just filtering explicitly at the leaves like here)

Reply:
It was not. It worked here due to that we have a bigger switch in experimental for all hip hal stuff I think. But I dropped the `SEND_ERROR` in #16510 so it should be fine now I believe. Though I still prefer to keep these tests in experimental directory together with hip hal just right now given they are not actually all passing at the moment.